app/src/main/java/org/oppia/android/app/profile/AdminPinActivityPresenter.kt

```diff
@@ -48,43 +47,27 @@ class AdminPinActivityPresenter @Inject constructor(
       viewModel = adminViewModel
     }
 
-    binding.inputPin.post {
-      addTextChangedListener(binding.inputPin) { pin ->
-        pin?.let {
-          adminViewModel.pinErrorMsg.set("")
-          adminViewModel.savedPin.set(it.toString())
-          inputtedPin = pin.isNotEmpty()
-          setValidPin()
-        }
+    binding.adminPinInputPinEditText.onTextChanged { pin ->
```

Comment:
Add a kdoc for this 
Ref - [kdoc for onTestChanged](https://github.com/anandwana001/oppia-android/blob/462f7fcab7637b0fe1083a254863cf32f528c60b/app/src/main/java/org/oppia/android/app/profile/AdminAuthActivityPresenter.kt#L49)

Reply:
Done